stories/action-sheet/action-sheet.stories.js

```diff
@@ -0,0 +1,200 @@
+import '../../dist/avatar.css';
+import '../../dist/icon.css';
+import '../../dist/action-sheet.css';
+
+export default {
+    title: 'Components/Action Sheet',
+    parameters: {
+        description: `
+An action sheet consists of a list of options a user can select from to complete an action. Actions can be
+clustered if there is not enough space on the screen
+            
+## Usage
+
+**Use the action sheet if:**
+
+- You need an option that provides more than one action.
+- It is really important that the user stays in context on a phone.
+- You only have a small number of actions.
+
+**Do not use the action sheet if:**
+
+- The menu provides only one option. In this case, consider using a button instead.
```

Comment:
I capitalized it and made it bold. I'm not sure if there is easy way to include links, I would need to investigate it and address this issue in other components as well, because we have a lot of appearance of external components. Maybe it can be skipped for now